### PR TITLE
Add a new microservices framework light-java

### PR DESCRIPTION
**Pull request by stevehu**

### CODE REVIEW

**Comment by nbrady-techempower** [comment]
@stevehu Thanks for your submission! It looks like there's a lot of extra stuff in here, like scripts to fill the databases. That stuff should be removed and made sure that we're using the `TFB-database`

**Comment by stevehu** [comment]
These extra files support local development and I only test within the vagrant with TFB-database. I will remove these file into a branch in case something needs to be changed later. Thanks.

**Comment by greenlaw110** [comment]
@stevehu I think your `UpdatesPostgresqlGetHandler` is using MySQL database by mistake:
image

**Comment by stevehu** [comment]
@greenlaw110 Thanks a lot for your help with the copy/paste error.

**Comment by stevehu** [comment]
@nbrady-techempower I have removed some files. Please let me know if you have questions. Thanks.

**Comment by greenlaw110** [comment]
@stevehu no worries. I found your mysql test doesn't work on my local environment. That's why I spot the copy/paste error.

BTW I think you need to update https://github.com/TechEmpower/FrameworkBenchmarks/blob/master/.travis.yml and add light-java inside. Otherwise the CI will not run your test.

Make sure you read http://frameworkbenchmarks.readthedocs.io/en/latest/Development/Add-Frameworks-or-Tests/

**Comment by stevehu** [comment]
@greenlaw110 Thanks for the reminder. I have added an entry in .travis.yml

File: frameworks/Java/light-java/setup.sh [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+sed -i 's|DATABASE_HOST|'"${DBHOST}"'|g' src/main/resources/config/mysql.json
+sed -i 's|DATABASE_HOST|'"${DBHOST}"'|g' src/main/resources/config/postgres.json
```
Comment (nbrady-techempower) [inline frameworks/Java/light-java/setup.sh]: We're trying to get everyone to move away from doing `sed` statements here and replacing all `localhost` urls with `TFB-database`. There are entries in the `etc/hosts` file

**Comment by stevehu** [comment]
@nbrady-techempower I have updated teh setup.sh and hard coded the database host in mysql.json and postgres.json. Please review and let me know if you have questions. Thanks.

**Comment by nbrady-techempower** [comment]
Thanks for taking care of those things @stevehu 
This looks good to me, ping @msmith-techempower for a second look

**Comment by msmith-techempower** [comment]
LGTM!

**Pull request merged**